{"info":"nova vulnerability","CVEs":["CVE-2012-3371"],"usn":"USN-1501-1","threat":["An authenticated attacker could use this to cause a denial of service.."],"date":"July 11, 2012","special":"The problem can be corrected by updating your system to the following package versions:\nIn general, a standard system update will make all the necessary changes.","update":{"12.04":[{"package":"python-nova","version":"2012.1+stable~20120612-3ee026e-0ubuntu1.2"}]}}